kernel/net.c

```diff
@@ -94,6 +78,28 @@ int NCDInit(void)
 	return 0;
 }
 
+void GetMacAddress(void) {
```

Comment:
I think I would call this function `InitMacAddress` and I'd just call it once from somewhere in main.
![image](https://user-images.githubusercontent.com/1534726/76003242-c5ee5f80-5ed5-11ea-8ce8-f244fdb9ba31.png)